packages/core/src/internal/decorators/i18n.spec.ts

```diff
@@ -48,5 +48,8 @@ describe('i18n decorator', () => {
     component.setAttribute('cds-i18n', `{ "close": "${closeText}" }`);
     await componentIsStable(component);
     expect(component.i18n.close).toEqual(closeText);
+    component.setAttribute('cds-i18n', `{ "close": "ohai" }`);
+    await componentIsStable(component);
+    expect(component.i18n.close).toEqual('ohai', 'double set i18n');
```

Comment:
I added this test to force a reset of the i18n string. The test passes but, as noted below, this functionally does not work in all cases.

Reply:
updated. works in all cases for now.